### PR TITLE
[Consumption] Fix `meterId` empty format issue

### PR DESCRIPTION
**Pull request by dvbb**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by dvbb** [comment]
Linked issue: https://github.com/Azure/azure-sdk-for-net/issues/36876

File: sdk/consumption/Azure.ResourceManager.Consumption/src/autorest.md [hunk 1]
```diff
@@ -88,6 +85,13 @@ rename-rules:
   URI: Uri
   Etag: ETag|etag
 
+models-to-treat-empty-string-as-null:
+  - ConsumptionLegacyReservationRecommendation
+  - ConsumptionLegacyUsageDetail
+  - ConsumptionMarketplace
+  - ConsumptionModernReservationRecommendation
+  - ConsumptionModernUsageDetail
```
Comment (live1206) [inline sdk/consumption/Azure.ResourceManager.Consumption/src/autorest.md:93]: How about `PriceSheetProperties`? It also contains `MeterId`

Comment (dvbb) [reply sdk/consumption/Azure.ResourceManager.Consumption/src/autorest.md:93]: resolved

**Review by live1206 — APPROVED** [review]
Please update the changelog with this bug fix.

**Review by archerzz — APPROVED** [review]

**Pull request merged**